assets/pesy-package.template.json

```diff
@@ -13,13 +13,17 @@
   "buildDirs": {
     "test": {
       "require": ["<PUBLIC_LIB_NAME>"],
-      "bin": "Test<PACKAGE_NAME_UPPER_CAMEL>.re as Test<PACKAGE_NAME_UPPER_CAMEL>.exe"
+      "bin": {
+        "Test<PACKAGE_NAME_UPPER_CAMEL>.exe": "Test<PACKAGE_NAME_UPPER_CAMEL>.re"
+      }
     },
     "library": {
     },
     "executable": {
       "require": ["<PUBLIC_LIB_NAME>"],
-      "bin": "<PACKAGE_NAME_UPPER_CAMEL>App.re as <PACKAGE_NAME_UPPER_CAMEL>App.exe"
+      "bin": {
+        "<PACKAGE_NAME_UPPER_CAMEL>App.exe": "<PACKAGE_NAME_UPPER_CAMEL>App.re"
```

Comment:
Does `.exe` mean we get this extension even on Linux/macOS? Can we omit it and have automatically added (by `dune` I think) on Windows.

Reply:
Yes. But, detecting where a package is a lib or bin still requires `.exe` extension. So removing this will require removing those code paths too. I'll do this in next PR if it okay.

Reply:
Isn't having `"bin"` enough to decide on lib vs bin?

Reply:
Yes it is. And the fix is trivial really. I need to update checks like [this](https://github.com/esy/pesy/blob/master/library/PesyConf.re#L983)